src/metabase/driver.clj

```diff
@@ -675,6 +675,9 @@
     ;; many databases in it.
     :connection/multiple-databases
 
+    ;; Does this driver support UUID type
+    :uuid-type
```

Comment:
You should mention new feature flags in the driver changelog file so third-party driver authors can add them if appropriate.